needle.pm

```diff
@@ -125,26 +127,6 @@ sub new ($classname, $jsonfile) {
     return $self;
 }
 
-sub save ($self, $fn = $self->{file}) {
```

Comment:
Are you removing this function because it is unused? I'm wondering whether this function is really completely useless (e.g. it might be useful during development/debugging) but I'm also not against removing it (as the function is not very well written anyway).

Reply:
See https://progress.opensuse.org/issues/167932#note-11 where I investigated if the function is used.
Note that it already got a bug 5 years ago.

When you say "useful during development/debugging", what do you mean by that? During openQA development, or during developing an test openQA test? When and how would you use it?

Reply:
During development of os-autoinst itself to dump a needle for debuging purposes. But as I said, I'm fine with removing the function.

Reply:
I would just use `YAML::PP::DumpFile('file.yaml', $needle)` if I wanted to debug the content into a file...